pkgs/by-name/ho/hoppscotch-community-edition/package.nix

```diff
@@ -0,0 +1,122 @@
+{
+  fetchFromGitHub,
+  lib,
+  makeBinaryWrapper,
+  nodejs,
+  pnpm,
```

Comment:
Most likely you need to change `pnpm` to `pnpm_9`. Since #371832, it points to `pnpm_10` instead of `pnpm_9`.

Reply:
They seem to support pnpm v6+, so this _should_ be fine? [tm]
https://docs.hoppscotch.io/documentation/self-host/community-edition/prerequisites#pnpm

I've asked the upstream just to make sure anyway.

Reply:
If pnpm 10 works please use that. I've seen it work for other people too, maybe I was unlucky when picking a few packages and none of them supporting 10 when I tested it.

Reply:
Got a reply back from upstream. They use whatever is the latest on the `npm` registry; and as of writing, `pnpm` is still 9.15.4. So, thanks for the review! Though it'll be a while until I revisit packaging this, when I do so I'll address this.

Reply:
The new [`2025.1.1`](https://github.com/hoppscotch/hoppscotch/releases/tag/2025.1.1) release uses pnpm v10.2.1, so I'll resolve this review.